internal/usage/usage.go

```diff
@@ -143,10 +143,12 @@ dbName and collection are required only for built-in roles.`
 	OrgNameFilter                             = "Organization name to perform a case-insensitive search for."
 	OrgIncludeDeleted                         = "Flag that indicates whether to include deleted organizations in the list. This option applies only to Ops Manager organizations. You can't return deleted Atlas or Cloud Manager organizations."
 	APIKeyDescription                         = "Description of the API key."
-	AtlasAPIKeyDescription                    = APIKeyDescription + requiredForAtlas
-	APIKeyRoles                               = "Role or roles that you want to assign to the API key. To assign more than one role, specify each role with a separate role flag or specify all of the roles as a comma-separated list using one role flag. To learn which values the CLI accepts, see the Items Enum for roles in the Atlas API spec: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Programmatic-API-Keys/operation/createApiKey."        //nolint:gosec // This is just a message not a credential
-	ProjectAPIKeyRoles                        = "Role or roles that you want to assign to the API key. To assign more than one role, specify each role with a separate role flag or specify all of the roles as a comma-separated list using one role flag. To learn which values the CLI accepts, see the Items Enum for roles in the Atlas API spec: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Programmatic-API-Keys/operation/createProjectApiKey." //nolint:gosec // This is just a message not a credential
-	AtlasAPIKeyRoles                          = APIKeyRoles + requiredForAtlas
+	AtlasAPIKeyDescription                    = APIKeyDescription + " " + requiredForAtlas
+	CreateProjectAPIKeyRoles                  = "Role or roles that you want to assign to the API key. To assign more than one role, specify each role with a separate role flag or specify all of the roles as a comma-separated list using one role flag. For the full list of accepted values, see the Items Enum for the corresponding :oas-atlas-tag:`Atlas API endpoint </Programmatic-API-Keys/operation/createProjectApiKey>`. To learn more about project level user roles, see :atlas:`/reference/user-roles/#project-roles`."    //nolint:gosec // This is just a message not a credential
```

Comment:
I believe we should hard-code this link as this message appears in the CLI help text